client/components/Cart.jsx

```diff
@@ -0,0 +1,89 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import {default as LineItem} from './LineItem.jsx'
+import { fetchOrder, removeLineItem, checkoutOrder, changeQuant } from '../store'
+
+function Cart (props) {
+  const { order, loading, handleClick, handleCheckout, handleQuantityChange } = props
+
+  if (order.lineItems){
+    var lineItems = order.lineItems
+    var totalPrice = lineItems.map((lineItem) => lineItem.getTotal * 1).reduce(function (a, b) {
```

Comment:
Question? Why did we do multiply 1 times lineItem.getTotal?

Reply:
getTotal returns a string instead of a number and I coerced it to a number with * 1 in order to reduce it otherwise instead of doing 1+2+3  = 6 it gives me 123